### PR TITLE
Gitignore improvements

### PR DESCRIPTION
**Pull request by alexanderkjall**
Deleted a checked in binary file that I'm guessing shouldn't be there `cmd/vulcan-aws-trusted-advisor/vulcan-aws-trusted-advisor`.

Added gitignore rules for all binaries generated by the checks.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,6 @@
 *.swp
 local.toml
+
+# editor files
+.idea/
+vulcan-checks.iml
```
Comment (julianvilas) [inline .gitignore:6]: Could you and a newline at the end of the file? Helpful when running bash commands/scripts :)

**Review by julianvilas — CHANGES_REQUESTED** [review]
Just a minor comment, for the rest looks super good to me :)

File: .gitignore [hunk 2]
```diff
@@ -1,2 +1,6 @@
 *.swp
 local.toml
+
+# editor files
```
Comment (jesusfcr) [inline .gitignore:4]: And this too just in case:

`.vscode/`

File: .gitignore (hunk 1 above)
Comment (alexanderkjall) [reply .gitignore]: fixed :)

File: .gitignore (hunk 2 above)
Comment (alexanderkjall) [reply .gitignore:4]: Added :)

**Review by julianvilas — APPROVED** [review]

**Pull request merged**